AudioKit/Tables/Generators/AKExponentialTableGenerator.m

```diff
@@ -46,12 +46,12 @@ - (NSArray *)parametersWithSize:(NSUInteger)size
         return [obj1[0] compare: obj2[0]];
     }];
     
-    int maximumIndex = (int)[[points lastObject] objectAtIndex:0];
+    NSInteger maximumIndex = [[[points lastObject] objectAtIndex:0] integerValue];
     float scalingFactor = (float)size/(float)maximumIndex;
     
     NSMutableArray *scaledPoints = [[NSMutableArray alloc] init];
     for (NSArray *point in points) {
-        int index = (int)[point objectAtIndex:0];
+        NSInteger index = [[point objectAtIndex:0] integerValue];
```

Comment:
Good call by the way, that old code must have been causing all sorts of bad shit, casting a pointer to an `int`. Hope there's not more of this in other places...